config.default.yaml

```diff
@@ -3,12 +3,26 @@
 # SPDX-License-Identifier: CC0-1.0
 
 version: 1.0.0
+
 tutorial: false
 
 logging:
   level: INFO
   format: "%(levelname)s:%(name)s:%(message)s"
 
+countries: ["NG", "BJ"]
+# Can be replaced by country ["NG", "BJ"], continent ["Africa"] or user-specific region, see more at https://pypsa-earth.readthedocs.io/en/latest/configuration.html#top-level-configuration
+
+enable:
+  retrieve_databundle: true  #  Recommended 'true', for the first run. Otherwise data might be missing.
+  retrieve_cost_data: true  # true: retrieves cost data from technology data and saves in resources/costs.csv, false: uses cost data in data/costs.csv
+  download_osm_data: true  # If 'true', OpenStreetMap data will be downloaded for the above given countries
+  build_natura_raster: false  # If True, than an exclusion raster will be build
+  build_cutout: false
```

Comment:
For me, that is a great re-ordering! Hopefully, it'll make less likely unintentional data overwriting :)

Reply:
Thank you! I will keep it this way then :)